std/algorithm/iteration.d

```diff
@@ -66,31 +63,14 @@ import std.functional; // : unaryFun, binaryFun;
 import std.range.primitives;
 import std.traits;
 
-///
 template aggregate(fun...) if (fun.length >= 1)
 {
-    /**
```

Comment:
Maybe just remove the ddoc style here and let the text in place? The text is not wrong, it just is not finalized yet.